KonyAccelerate/controllers/mobile/frmFeedbackController/frmFeedbackController.js

```diff
@@ -0,0 +1,131 @@
+define({
+
+    //Type your controller code here 
+    selectedImage: "starselected.png",
+    unselectedImage: "starunselected.png",
+    objectServiceL: "KonyAccelerateStorageService",
```

Comment:
Use Constants in global constants file

Reply:
If we do that, it will become global. I want the scope to be specific to form controller.

Reply:
That will be useful if we want to use that in some other form

Reply:
Moved to Constants file.